src/partition/repart.c

```diff
@@ -977,7 +977,7 @@ static int config_parse_label(
         /* Nota bene: the empty label is a totally valid one. Let's hence not follow our usual rule of
          * assigning the empty string to reset to default here, but really accept it as label to set. */
 
-        r = specifier_printf(rvalue, specifier_table, NULL, &resolved);
+        r = specifier_printf(rvalue, 36, specifier_table, NULL, &resolved);
```

Comment:
maybe export the 36 as new define in gpt.h, i.e. GPT_LABEL_MAX or so, and then use here and in gpt_partition_label_valid()